core/src/main/scala/org/scalarelational/model/ModularSupport.scala

```diff
@@ -11,19 +12,28 @@ import org.scalarelational.instruction._
  *
  * @author Matt Hicks <matt@outr.com>
  */
-trait ModularSupport extends SQLContainer with Listenable with DataTypes {
+trait ModularSupport extends SQLContainer {
+  case class Processor[T]() {
+    val functions = ArrayBuffer.empty[T => T]
+    def fire(value: T): T =
```

Comment:
I think we should have a separate design discussion regarding Properties and Processors. So far I see this PR making drastic changes simply for the sake of removing powerscala while I see absolutely no advantage and several disadvantages.